### ISSUE TITLE
error

### ISSUE THREAD

**Issue by lnked** [avocode/avocode#1142]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.11.0`
Source UID: 93054
```
TypeError: Cannot read property '_currentElement' of null
    at Object.unmountComponent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconciler.js:51:59)
    at ReactCompositeComponentWrapper.unmountComponent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactCompositeComponent.js:246:21)
    at Object.unmountComponent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactReconciler.js:52:22)
    at Object.unmountChildren (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactChildReconciler.js:115:25)
    at ReactDOMComponent.unmountChildren (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactMultiChild.js:398:28)
    at ReactDOMComponent.unmountComponent (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/ReactDOMComponent.js:911:10)
    at Object.unmountComponent (/Applications/Avocode.app/Contents/Re
```

**Comment by connor-bean** [comment]
Hey, @lnked -

Can you provide us with more details on what you were doing when you received this error?

Thanks,
Connor, Avocode Support

**Comment by connor-bean** [comment]
Closing due to inactivity. Please reopen if you can provide more information.

Thanks,
Connor, Avocode Support

**Comment by petrbrzek** [comment]
FYI: this bug was fixed in 2.11.1. So just update your app and it should work.
